core/src/main/java/org/mskcc/cbio/portal/util/ValuesUtils.java

```diff
@@ -0,0 +1,36 @@
+package org.mskcc.cbio.portal.util;
+
+public class ValuesUtils {
+    public static enum MissingValues {
+        NOT_APPLICABLE("Not Applicable"), NOT_AVAILABLE("Not Available"), PENDING("Pending"),	
```

Comment:
for consistency would you mind putting these enum types on their own separate lines?